interpreter/interpreter.go

```diff
@@ -624,14 +624,14 @@ func (itrp *Interpreter) doCall(ctx context.Context, call *semantic.CallExpressi
 	f := callee.Function()
 
 	// Check if the function is an interpFunction and rebind it.
-	if af, ok := f.(function); ok {
-		af, err = itrp.mutateFunctionScope(af)
-		if err != nil {
-			return nil, err
-		}
-		af.itrp = itrp
-		f = af
-	}
+	// if af, ok := f.(function); ok {
+	// 	af, err = itrp.mutateFunctionScope(af)
+	// 	if err != nil {
+	// 		return nil, err
+	// 	}
+	// 	af.itrp = itrp
+	// 	f = af
```

Comment:
There are no objects that exist that need to be rebound. The objects that were created are newly created for every invocation. So instead of attempting to copy the package to avoid changing the cached interpreter packages, the interpreter package is unique to that run.